src/lib/email/create-calendar-invite.ts

```diff
@@ -16,7 +16,7 @@ export const createCalendarInvite = (
   params: CreateCalendarInviteParams,
 ): ICalCalendar => {
   const calendar = ical({
-    name: "De Sering",
+    name: process.env.ORG_NAME + "Calendar" || "Event Calendar",
```

Comment:
I think that's missing a space
```suggestion
    name: process.env.ORG_NAME + " Calendar" || "Event Calendar",
```